src/db/db_build.sql

```diff
@@ -0,0 +1,81 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users, posts, messages, chatRooms, diseases, tags, comments;
+
+CREATE TABLE diseases (
+    id SERIAL PRIMARY KEY,
+    diseaseName VARCHAR(100) NOT NULL
+);
+
+INSERT INTO diseases (diseaseName)
+VALUES ('cancer');
+
+CREATE TABLE tags (
+    id SERIAL PRIMARY KEY,
+    tagName VARCHAR(50) NOT NULL,
+    diseaseId INTEGER REFERENCES diseases(id)
```

Comment:
can you give me an explanation on what ` INTEGER REFERENCES diseases(id)` would do ?

Reply:
and whats the difference between it and `FOREIGN KEY`?

Reply:
I don't have a better answer than what Finn said and I trust him.
I can change it if it is important to you ^-^

Reply:
> I don't have a better answer than what Finn said and I trust him.

i don't :laughing: 
but if it behaves like foreign key and **it works** then its fine by me

Reply:
:+1: